src/targets/gpu/mlir.cpp

```diff
@@ -1081,12 +1134,21 @@ tuning_config get_tuning_config_mlir(const context& migraphx_ctx,
                                      bool exhaustive)
 {
     adjust_param_shapes(m, inputs);
-
+    rewrite_reduce(m);
     mlir_program mp;
     mp.set_gpu_properties(migraphx_ctx);
     mp.parse(m);
     auto tc = mp.get_tuning_config(exhaustive);
-
+    std::string problem_config = tc.problem.to<std::string>();
+    for(const auto i : iterator_for(m))
+    {
+        if(starts_with(i->name(), "@"))
+        {
+            continue;
+        }
+        problem_config += " " + i->name();
+    }
+    tc.problem       = problem_config;
```

Comment:
I had these changes to experiment and work around problem_cache issue. Reverting these